phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/Repository.java

```diff
@@ -38,6 +38,7 @@ default void fetchInstallableApplications(Consumer<RepositoryDTO> callback, Cons
         try {
             callback.accept(fetchInstallableApplications());
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
I think it might be a good idea to add a custom `RepositoryException` class which can be thrown inside repositories and is used as a wrapper for other exceptions.
This will enable us to specify behavior that should throw an exception, which leads to an error screen, and other problems that should terminate the application (like out of memory or other bad things).